js/templates/changeServerWarningModal.html

```diff
@@ -0,0 +1,10 @@
+<h1>Please Note Settings</h1>
```

Comment:
I haven't poly-glotted this guy, since I imagine it still might change. If we settle on it, at least for a while, I'd be happy to poly-glot it.